### ISSUE TITLE
Zwift_push gh action is still failing

### ISSUE THREAD

**Issue by netbrain** [netbrain/zwift#112]
+ docker cp zwift:/bin/run_zwift.sh /tmp/run_zwift.sh
Error response from daemon: Could not find the file /bin/run_zwift.sh in container zwift

https://github.com/netbrain/zwift/actions/runs/8968933628

https://github.com/netbrain/zwift/actions/runs/8972028542

**Comment by sHedC** [comment]
Is this because the latest docker file is still not containing the refector stuff?
I see the full build failed, my push should resolve that as I got that when testing build with docker so added a fix.

**Comment by netbrain** [comment]
Yes probably. I'm currently sick 😷 but as soon I'm on my feet I'll try to review all the things. 🤞👍

**Comment by netbrain** [comment]
Seems to be working now
